spec/acceptance/00_backend_spec.rb

```diff
@@ -92,6 +92,42 @@ class { 'sensu::backend':
     end
   end
 
+  context 'etc_dir change' do
+    it 'should work without errors' do
```

Comment:
To ensure that the old, default is not being used, perhaps add something like

```puppet
file { '/etc/sensu':
  ensure  => absent,
  recurse => true,
}
```

Reply:
That won't tell us anything that isn't already being tested. The test for port change configs are how we test this works.  We modify the `api-port` via the YAML config.  If that YAML config was not read from new location the port would not get changed.  Testing the new config gets used is a more direct test anyways.

Reply:
add a comment that explains this

Reply:
Comments added, also to agent acceptance tests that test the same thing.